index.html

```diff
@@ -13,58 +13,70 @@
 
 <body>
     <div class="container">
+        <header>
+            <nav class="top-nav">
+                <img class="logo" src="img/lambda-black.png" alt="Lambda">
+                <a href="#">Home</a>
+                <a href="about.html">About</a>
+                <a href="products.html">Products</a>
+                <a href="#">Blog</a>
+                <a href="#">Contact</a>
+            </nav>
+        </header>
    
-        <section class="top-content">
-            <div class="text-container">
+        <img class="main-img" src="img/jumbo.jpg" alt="Warp Speed!">
+
+        <section class="index-top">
+            <div class="index-top-para">
                 <h2>The Future</h2>
                 <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
-            <div class="text-container">
+            <div class="index-top-para">
                 <h2>The Past</h2>
                 <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
         </section>
         
-        <section class="middle-content">
+        <section class="index-main">
         
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box" id="box1">Box 1</div>
+                <div class="box" id="box2">Box 2</div>
+                <div class="box" id="box3">Box 3</div>
+                <div class="box" id="box4">Box 4</div>
+                <div class="box" id="box5">Box 5</div>
+                <div class="box" id="box6">Box 6</div>
+                <div class="box" id="box7">Box 7</div>
+                <div class="box" id="box8">Box 8</div>
+                <div class="box" id="box9">Box 9</div>
+                <div class="box" id="box10">Box 10</div>
```

Comment:
Be careful with the use of ids in your code. Remember, ids help declare absolute specificity.  In this instance, I would have used a simple `<div>` distinction between the boxes (i.e. `<div class="box 1">`,  etc).